src/Basque/Mews.Fiscalizations.Basque/Dto/Bizkaia/Header/BizkaiaHeaderData.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Mews.Fiscalizations.Basque.Dto.Bizkaia.Header;
+
+public class BizkaiaHeaderData
+{
+    [JsonPropertyName("con")]
+    public string With { get; set; }
+    
+    [JsonPropertyName("apa")]
+    public string Section { get; set; }
+
+    [JsonPropertyName("inte")]
+    public Inte Inte { get; set; }
+
+    [JsonPropertyName("drs")]
+    public Drs Drs { get; set; }
+}
+
+public class Drs
+{
+    [JsonPropertyName("mode")]
+    public string Mode { get; set; }
+
+    [JsonPropertyName("ejer")]
+    public string FiscalYear { get; set; }
+}
+
+public class Inte
+{
+    [JsonPropertyName("nif")]
+    public string TaxPayerIdentificationNumber { get; set; }
+
+    [JsonPropertyName("nrs")]
+    public string Name { get; set; }
```

Comment:
if i remember correctly this is either the first name or the company name, pls double check
and if yes, i'd go with something like FirstNameOrCompanyName